API-Samples/C#/ArmBased/VideoIndexerClient/auth/AccessTokenRequest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Text.Json.Serialization;
+
+namespace VideoIndexingARMAccounts.VideoIndexerClient.auth
```

Comment:
change to `VideoIndexingARMAccounts.VideoIndexerClient.Auth` (PascalCase)

Reply:
done